artifactory/commands/transferfiles/utils.go

```diff
@@ -295,6 +295,7 @@ func handleChunksStatuses(phase *phaseBase, chunksStatus *UploadChunksStatusResp
 }
 
 func updateProgress(phase *phaseBase, progressbar *TransferProgressMng, timeEstMng *timeEstimationManager, chunk ChunkStatus, workingThreads int) error {
```

Comment:
Let'a add a description for this fuunction.

Reply:
Done!